Registration/DRR Generation/Python/DRR.py

```diff
@@ -0,0 +1,161 @@
+# This script generates Digitally Reconstructed Radiograph (DRR) from a given 
+# 3D image to given 2D image plane(s)
+
+# ISSUES - 
+# 1- I don't know how to set the direction of the output image
+
+import itk # imports insight Toolkit
+import numpy as np
+import sys
+from Functions import ChangeImageDirection
+
+
+
+#%% -------------------------------- Generated Inputs --------------------------------- 
+direction = np.identity(3,dtype=np.double)
+#direction[0,:] = [np.sqrt()/2,-np.sqrt(1)/2,0.] # Creating a non identity direction output just for testing. 
+#direction[1,:] = [np.sqrt(1)/2, np.sqrt(3)/2,0.] # Creating a non identity direction output just for testing. 
+
+
+#%% -------------------------------- INPUTS --------------------------------- 
+input_filename  = '/Volumes/Storage/Projects/Registration/QuickData/OA-BEADS-CT.nii'
```

Comment:
It's ususally not good to hard code your input file names. It is preferable that you set them as command line arguments. I can clean this up later though.

Reply:
I agree ... I am going to change it today by using xtine's Classes. But I need to close my current branch before I can branch out to have xtine's header reading stuff.

Reply:
Okay, awesome!